plutus-playground-client/src/Action.purs

```diff
@@ -4,6 +4,7 @@ module Action
   ) where
 
 import Types
+
```

Comment:
why is this by itself?

Reply:
Oh, that'll be purs:ide's idea. I think it likes to put a space after the one-blessed-unqualified import (usually prelude).